docs/argocd-gitops/onboarding_to_argocd.md

```diff
@@ -1,33 +1,52 @@
 # ArgoCD onboarding procedure
 
-The ArgoCD instance is deployed on [MOC Infra][1] cluster and can be used to deploy the application manifests located in this repo.
+We have an ArgoCD instance deployed on MOC that can be used to deploy OpenShift resources located on a Git Repository.
 
-Each team is given an [ArgoCD Project][2] that has an allow-list of clusters and namespaces to which they can deploy. They also have an allow-list of resources they can deploy within those namespaces. These restrictions exist to prevent teams from being able to use ArgoCD to deploy cluster scoped resources, or resources onto other team's namespaces. You can browse the projects [here][3] to get an idea of the current set of permissions. Note that for most projects, majority of these permissions are inherited from the [global project][4].
+The ArgoCD instance is deployed on [MOC Infra][1] cluster.
+
+Each team is given an [ArgoCD Project][2] that has an allow-list of clusters and namespaces to which they can deploy.
+They also have an allow-list of resources they can deploy within those namespaces. These restrictions exist to prevent
+teams from being able to use ArgoCD to deploy cluster scoped resources, or resources onto other team's namespaces. You
+can browse the projects [here][3] to get an idea of the current set of permissions. Note that for most projects,
+majority of these permissions are inherited from the [global project][4].
```

Comment:
Should we add something about the process one would go through if they wanted to adjust these permissions, such as if they were deploying an application that required them to enable a new type of CRD in the global project?

Reply:
Like this? https://github.com/operate-first/apps/blob/master/docs/argocd-gitops/add_permissions_to_project.md

Reply:
Yeah, good to know its already addressed.